ledger/src/blockstore.rs

```diff
@@ -170,6 +174,31 @@ impl<T> AsRef<T> for WorkingEntry<T> {
     }
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct LastFECSetCheckResults {
+    block_id: Option<Hash>,
```

Comment:
Is there a place this `block_id` is defined ?
Would it make sense to call this `last_fec_set_merkle_root` or something similar to be more expressive?

Reply:
https://github.com/anza-xyz/agave/blob/a2eaf6d189a4316c6f5ef8d9508dc628c7548e7b/sdk/program/src/vote/state/mod.rs#L235-L238
This is where it is defined. I'll keep it as `last_fec_set_merkle_root` here and we can start calling it `block_id` in the replay pipeline.